lib/app.js

```diff
@@ -1,18 +1,27 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
+const cors = require('cors');
 
-const authenticate = require('./middleware/authenticate');
+// const authenticate = require('./middleware/authenticate');
 
 const app = express();
 
 // Built in middleware
 app.use(express.json());
 app.use(cookieParser());
-
-// App routes
+app.use(
+  cors({
+    origin: [
+      'https://alchemy-shopping-front-end-demo.netlify.app',
+      'http://localhost:5501',
+    ],
+    credentials: true,
+  })
+);
+// App routes //
 
 app.use('/api/v1/users', require('./controllers/users'));
-
+app.use('/api/v1/items', require('./controllers/items'));
```

Comment:
`app.use('/api/v1/items', authenticate, require('./controllers/items'));`

this would apply the middleware to all the routes